packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -62,7 +59,15 @@ export const Default = {
 };
 
 /**
- * Use the `variant` attribute to display the brandshape with different background colors.
+ * Use the `variant` attribute to set the color variant:
+ *
+ * - `primary` (default): used on light backgrounds
+ * - `neutral-100`
+ * - `border-primary`: used on light backgrounds
+ * <br>Change the fill color to match the background color
+ * - `image`
+ * - `white`: used on primary, primary-100 and neutral-100 backgrounds
```

Comment:
```suggestion
 * - `white`: used on primary, `primary-100` and `neutral-100` backgrounds
```

Reply:
In this case, me and @MartaPintoTeixeira believe it shouldn't be highlighted because it is not a class or a value. Therefore, it serves as a way to differentiate. We think the highlight may mislead the user into believing it is also a class.